discord/guild.py

```diff
@@ -647,7 +647,7 @@ def me(self) -> Member:
         return self.get_member(self_id)  # type: ignore
 
     @property
-    def voice_client(self) -> VoiceProtocol | None:
+    def voice_client(self) -> VoiceClient | None:
         """Returns the :class:`VoiceProtocol` associated with this guild, if any."""
```

Comment:
If you're updating typehints, update the associated docstrings too.
```suggestion
        """Returns the :class:`VoiceClient` associated with this guild, if any."""
```